xtrlock.man

```diff
@@ -48,3 +48,15 @@ This program is distributed in the hope that it will be useful, but
 WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 General Public License for more details.
+
+This program now allows user to use their own password instead of 
+the default user password, which is the sudo password (for many 
```

Comment:
"the sudo password" sounds weird. How about:
> instead of their login password, which often is also used for other things such as root access via ```sudo```.